src/App.test.jsx

```diff
@@ -1,4 +1,33 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import App from './App';
 
-test('Should render the header', () => {
+describe('Header Tests', () => {
+  it('Should render the header image', () => {
+    render(<App />);
 
-})
+    // test for header image
+    screen.getByAltText('Alchemy Logo');
+
+    // screen.debug()
+  });
+
+  it('Should render the header profile name', async () => {
```

Comment:
It is totally fine to test for all of the header elements in one header test. Nice job though!